mods/ra/rules/civilian.yaml

```diff
@@ -70,18 +70,31 @@ FCOM:
 		Dimensions: 2,3
 		LocalCenterOffset: 0,-512,0
 	Valued:
-		Cost: 2000
+		Cost: 1
```

Comment:
Neither 'givebounty', 'cost' or 'valued' was neccessary and depended on each other in a closed loop. Removed them all (as done with Oil Derric/Hospital).